lib/home/profile/account/change_password.dart

```diff
@@ -1,17 +1,219 @@
 import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:thc/utils/navigator.dart';
 
-class ChangePasswordScreen extends StatelessWidget {
+class ChangePasswordScreen extends StatefulWidget {
   const ChangePasswordScreen({super.key});
 
+  @override
+  // ignore: library_private_types_in_public_api
+  _ChangePasswordScreenState createState() => _ChangePasswordScreenState();
```

Comment:
Instead of ignoring this message, we can fix it!

```suggestion
  State<ChangePasswordScreen> createState() => _ChangePasswordScreenState();
```